cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -50,6 +50,7 @@
     if node.chef_environment == 'staging' && node.name == 'staging' # 'real' staging only
       cronjob at:'@reboot', do:home_dir('.dropbox-dist', 'dropboxd')
       cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'import_google_sheets')
+      cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'import_google_sheets_dashboard')
```

Comment:
this doesn't seem right -- the purpose of the existing `import_google_sheets` cronjob is to sync from gsheets to csv: https://github.com/code-dot-org/code-dot-org/blob/07ba656ae39c5be9fc9ec4b64b6b0b0694c35b77/pegasus/rake/seed.rake#L20-L27

since you are piggy backing off the existing csv file, the cronjob logic does not need to be replicated. (see next comment for what to do instead)